feat: Support svelte.config.ts (#13935)
sveltejs/kit: .changeset/twenty-pants-fetch.md

---

> '@sveltejs/package': patch
> '@sveltejs/kit': patch

Reviewer: Should this be a minor instead since it's new functionality?
```suggestion
'@sveltejs/package': minor
'@sveltejs/kit': minor
```

---

feat: support svelte.config.ts

> **NOTE**
>

> > Your runtime has to support importing TypeScript files for svelte.config.ts to work.

Reviewer 2: ```suggestion
> Your runtime has to support importing TypeScript files for `svelte.config.ts` to work.
```

> > In nodejs you can enable it with the `--experimental-strip-types` flag

Reviewer 2: I don't think you need the flag on the latest Node 24. https://github.com/nodejs/node/pull/56350

Author: it doesn't break if you still pass that flag or do you think we have to be specific and list each minor where it was added as a flag and then each that unflags it?

Reviewer 2: ```suggestion
> In Node.js, the feature is supported with the `--experimental-strip-types` flag starting in Node 22.6.0 and supported without a flag starting in Node 23.6.0.
```
